rtc_data_enclave/build.rs

```diff
@@ -7,6 +7,7 @@ use std::env;
 fn main() {
     println!("cargo:rerun-if-changed=Enclave.edl");
     println!("cargo:rerun-if-changed=src");
+    println!("cargo:rerun-if-changed=../codegen/data_enclave");
```

Comment:
This will rerun if the `bindings.h` file changed, causing the build script to always run twice if something changed (not a big issue, just noting it). 

Another thing, this might not behave in an expected manner since the codegen output is only generated after this build script runs (in the makefile). 

I might be misunderstanding how the `rerun-if-changed` cargo directive work.

Reply:
I checked, and it doesn't look like the build script will run twice.

However, on a closer look, it looks like this might unnecessarily rebuild things after changes to the `Enclave_{t,u}.{c,h}`, so we should probably limit this directive to just the `bindings.h` file itself.